cmd/root.go

```diff
@@ -107,6 +107,11 @@ func runE(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, args []string)
 		configFile = os.Getenv("TREEFMT_CONFIG")
 	}
 
+	// fallback to PRJ_ROOT
+	if configFile == "" {
+		configFile = os.Getenv("PRJ_ROOT")
```

Comment:
If $PRJ_ROOT is set, it should look for either $PRJ_ROOT/treefmt.toml or $PRJ_ROOT/.treefmt.toml. And error out if none of those are found.